src/amoc_distribution/amoc_cluster.erl

```diff
@@ -282,3 +280,8 @@ maybe_set_master(Node, #state{new_connection_action = Action}) ->
     %% to avoid a possibility of the amoc_cluster deadlock while
     %% running the Action call set_master_node/2 asynchronously
     spawn(fun() -> set_master_node(Node, Action) end).
+
+-spec execute_nodes(atom(), [node()], #{any() => any()}) -> ok.
+execute_nodes(Name, Nodes, State) ->
+    PrefixedName = [amoc, cluster, Name],
+    telemetry:execute(PrefixedName, #{count => length(Nodes)}, #{nodes => Nodes, state => State}).
```

Comment:
what is the point of having amoc_telemetry module but not using it? why do you want to have monotonic timestamps for some but not these event?

Reply:
I think that at this point I'm just confused with so much multi-tasking 🥲
Will fix in a sec.

Reply:
man, maybe better have a rest, it's friday evening, find some more attractive activity. I heard it also helps with multi-tasking